pkg/v5/configmap/desired.go

```diff
@@ -0,0 +1,74 @@
+package configmap
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/cluster-operator/pkg/label"
+)
+
+func (s *Service) GetDesiredState(ctx context.Context, configMapValues ConfigMapValues) ([]*corev1.ConfigMap, error) {
+	desiredConfigMaps := make([]*corev1.ConfigMap, 0)
+
```

Comment:
Desired state returns in the Ingress Controller config map.

Reply:
I see how this is working, do we plan on dealing with per guest cluster configuration through in the future? I don't see an easy way here to pass values on a per guest cluster basis from the outside. Sorry if this is off topic, but I wondered about this.

Reply:
Marcel, see my other comment. I think per guest cluster config will be handled as part of the Service Catalogue story.